fastplotlib/layouts/_figure.py

```diff
@@ -474,7 +474,7 @@ def show(
                     _maintain_aspect = subplot.camera.maintain_aspect
                 else:
                     _maintain_aspect = maintain_aspect
-                subplot.auto_scale(maintain_aspect=_maintain_aspect, zoom=0.95)
+                subplot.auto_scale(maintain_aspect=maintain_aspect)
```

Comment:
I removed the zoom so that the scaling is nice w.r.t. axes positions when you do `fig.show()`

Reply:
I think we should add a `bool` in `show()` that allows axes visibility to be `False` to start 

```python
fig.show(axes=False)
```